Composer/packages/client/src/pages/botProject/runtime-settings/RuntimeSettings.tsx

```diff
@@ -43,6 +45,14 @@ import {
 
 type RuntimeType = 'path' | 'command';
 
+/** Determine if a bot is configured to use the adaptive runtime vs the legacy runtime */
+export const isAdaptiveRuntime = (settings: DialogSetting): boolean => {
```

Comment:
@bfc/shared now has the isAdaptiveRuntime check you can use.

Reply:
updated!